src/go/rpk/pkg/cli/cmd/config.go

```diff
@@ -121,10 +121,10 @@ func bootstrap(fs afero.Fs) *cobra.Command {
 			conf.Redpanda.RPCServer.Address = ownIp.String()
 			conf.Redpanda.KafkaApi.Address = ownIp.String()
 			conf.Redpanda.AdminApi.Address = ownIp.String()
-			conf.Redpanda.SeedServers = []*config.SeedServer{}
-			seeds := []*config.SeedServer{}
+			conf.Redpanda.SeedServers = []config.SeedServer{}
```

Comment:
I think these changes belong in their own commit, so e6c0176b2850d8d4e6b297fbe68a1f8cd4222d00 is a clean move.
443c20bd6ca8bea29a42ea6ac7f998924ae20865 shows the way :)

Reply:
_Cries in rebase_
I missed this when I was organizing everything :cry: